source/management.rst

```diff
@@ -1,3 +1,14 @@
 =============================
  The Management of the PyCQA
 =============================
+
+- Creator: Ian Cordasco
+
+- Pep8 Administrator: Ian Lee
```

Comment:
Yep. Was wondering when you wanted to start that transition @IanLee1521. I am happy to be of service for whenever you want to schedule that. :)